ui/apps/platform/src/Containers/NetworkGraph/components/TimeWindowSelector.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Select, SelectOption } from '@patternfly/react-core';
+
+import { timeWindows } from 'constants/timeWindows';
+import useSelectToggle from 'hooks/patternfly/useSelectToggle';
+
+type TimeWindowSelectorProps = {
+    setActiveTimeWindow: (timeWindow) => void;
+    activeTimeWindow: string;
```

Comment:
Would typing these values with the same `typeof timeWindows[number]` used in the state of NetworkGraphPage cause problems with PF's Select component props?